RTL_testcases/EDA-1359/2D_FHT_raptor_project_high_fast_area_error/raptor_run.sh

```diff
@@ -10,8 +10,8 @@ ip_name="" #design_level
 tool_name="iverilog" 
 
 #simulation stages
-post_synth_sim=false 
-post_route_sim=false 
+post_synth_sim=true 
+post_route_sim=true 
```

Comment:
@zaheerahmad-rs what is this change ? Is it related to the script to be removed ?